README.md

```diff
@@ -52,21 +54,41 @@ The HelloID connector uses the API endpoints listed in the table below.
+| ----------- | ------ | ------ | ------ | ------- | ------ | --------------------- | -------------------------------------------------------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| Active      | X      | X      |        | X       |        | Yes                   | For create: Fixed: False<br>For enable: Fixed: True<br>For disable: Fixed: False | Yes       |                                                                                                                                |
+| Department  | X      |        | X      |         |        | Yes                   | Field: PrimaryContract.Department.DisplayName                                    | No        | Custom scripting in code to transform this to the corresponding scim object                                                    |
+| DisplayName | X      |        | X      |         |        | Yes                   | Complex: [displayName.js](mapping/displayName.js)                                | Yes       |                                                                                                                                |
+| Emails      | X      |        | X      |         |        | Yes                   | Complex: [emails.js](mapping/emails.js)                                          | Yes       | Custom scripting in code to transform this to the corresponding scim object                                                    |
+| ExternalId  | X      |        | X      |         |        | Yes                   | Field: ExternalId                                                                | Yes       |                                                                                                                                |
+| Manager     | X      |        | X      |         |        | Yes                   | None                                                                             | No        | Set within script, as the aRef of manager is used. Custom scripting in code to transform this to the corresponding scim object |
+| Title       | X      |        | X      |         |        | Yes                   | Field: PrimaryContract.Title.Name                                                | No        |                                                                                                                                |
+| Username    | X      |        | X      |         |        | Yes                   | Complex: [username.js](mapping/username.js)                                      | Yes       | Used for correlation                                                                                                           |
+
+### Correlation
```

Comment:
We use the username to correlate the account in Zenya. Is there also a possibility to include a PowerShell correlation report template? Kindof like the one in the Topdesk repository :)
Too bad there is no builtin option for this in HelloID.

Reply:
Technically, it is possible with powershell to create "our own" correlation report.
However, as Zenya is a SCIM connector, we can only query the users we have created or linked to with HelloID.
So before the actual create calls, there is no possibility to query all the users from Zenya and generate a report.

An option might be to manually create an export from Zenya and compare this to the evaluation of HelloID, but I'm not sure if that's possible within Zenya.

Reply:
agreed